components/omnivore/common/utils.mjs

```diff
@@ -0,0 +1,20 @@
+async function streamIterator(stream) {
+  const resources = [];
+  for await (const resource of stream) {
+    resources.push(resource);
+  }
+  return resources;
+}
```

Comment:
Suggestion: this could be a useful addition to the `@pipedream/helpers` package since many different apps use it